db/seeds.rb

```diff
@@ -25,6 +25,11 @@ def users_list
         email: "chinedu.dan@andela.com", street_address: "34, Adeyemo Alakija",
         city: "VI", state: "Lagos", password: "1234567890",
         user_type: "taskee", confirm_token: "112ewqee2123wqwqw12wq"
+      },
+       { firstname: "Chinedu", lastname: "Daniel",
+        email: "chinedu@disgui.se", street_address: "34, Adeyemo Alakija",
+        city: "VI", state: "Lagos", password: "qazwsxedc",
+        user_type: "tasker", confirm_token: "112ewqee2123wqwqw12wq", confirmed: true
```

Comment:
Line is too long. [84/80]<br>Align the elements of a hash literal if they span more than one line.